packages/router/__tests__/router-test.ts

```diff
@@ -9912,12 +9925,14 @@ describe("a router", () => {
       // Perform an invalid navigation to /parent/b which will be handled
       // using parent's error boundary.  Parent's deferred should be left alone
       // while A's should be cancelled since they will no longer be rendered
-      let formData = new FormData();
-      formData.append("file", new Blob(["1", "2"]), "file.txt");
-      await t.navigate("/parent/b", {
+      let B = await t.navigate("/parent/b", {
         formMethod: "get",
-        formData,
+        formData: new FormData(),
       });
```

Comment:
This can just become `let B = await t.navigate("/parent/b");` - no need for the submission aspect anymore

Reply:
Done